WireAPI/Sources/WireAPI/Components/ResponseParser.swift

```diff
@@ -30,10 +30,10 @@ struct ResponseParser<Success> {
         case noParseResult
     }
 
-    private typealias ParseBlock = (Int, Data?) throws -> Success?
+    typealias ParseBlock = (Int, Data?) throws -> Success?
 
     private let decoder: JSONDecoder
-    private var parseBlocks: [ParseBlock]
+    var parseBlocks: [ParseBlock]
```

Comment:
If you follow my previous suggestion about not using the response parser, this could go back to private, which I think creates a better api to the parser.

Reply:
changed